lib/ui/painting.dart

```diff
@@ -1951,8 +1951,8 @@ Float32List _encodeTwoPoints(Offset pointA, Offset pointB) {
 
 /// A shader (as used by [Paint.shader]) that renders a color gradient.
 ///
-/// There are two useful types of gradients, created by [new Gradient.linear]
-/// and [new Gradient.radial].
+/// There are three useful types of gradients, created by [new Gradient.linear]
+/// [new Gradient.radial], and [new Gradient.sweep].
 class Gradient extends Shader {
```

Comment:
Let's change this to something more general, like "There are several types of gradients, represented by the various constructors on this class."

That way we don't have to add something here the next time we add one. :-)

Reply:
Done.